tests/test_stats.py

```
@@ -0,0 +1,290 @@
+def test_long_track_full_confidence() -> None:
+    """Long iid track: T_eff >= 182, w = 1, sasr ≈ dasr."""
+    rng = np.random.default_rng(1)
+    values = 0.0005 + 0.008 * rng.normal(size=220)
+    metrics = rank_column(pd.Series(values, index=_daily_index(220)), RFR)
+    assert metrics.t_eff is not None and metrics.t_eff >= 182
+    assert metrics.w == pytest.approx(1.0)
+    assert metrics.sasr == pytest.approx(metrics.dasr)
+
+
+def test_positive_lag1_lowers_sasr_not_dasr() -> None:
+    """Same multiset: blocked (ρ>0) has lower T_eff and SASR; DASR unchanged."""
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `ρ` (GREEK SMALL LETTER RHO). Did you mean `p` (LATIN SMALL LETTER P)?

(RUF002)